syntaxes/haskell.YAML-tmLanguage

```diff
@@ -58,8 +58,8 @@ patterns:
         ^(\s*)\b(?:(?:(data)\s+(instance))|(data))
         \s+
         (
-          [\p{Lu}\p{Lt}][\w\p{Nd}_']*                   # named type
-        | \(\s*:[\p{S}\p{P}&&[^(),;\[\]`{}_"']]+\s*\)   # Operator type
+          [\p{Lu}\p{Lt}][\w\p{Nd}_']*             # named type
+        | \(:[\p{S}\p{P}&&[^(),;\[\]`{}_"']]+\)   # Operator type
```

Comment:
Is whitespace around parenthesized operators not allowed? I.e. `( :-> ) Type1 Tye2` is not a valid definition?

Reply:
Oh ok it is allowed, I was convinced it wasn't. I'll try to consistently allow spaces then, and include that in the tests. Thanks.

Reply:
Done.